### ISSUE TITLE
False negatives for disable-line block comments.

### ISSUE THREAD

**Issue by ota-meshi** [mysticatea/eslint-plugin-eslint-comments#41]
ESLint>=v4.18.0 supports `/* eslint-disable-line */` comments, but some rules in this plugin do not support this comments.

e.g.

```js
/*eslint eslint-comments/no-unlimited-disable: error */

/* eslint-disable-line */
```

```js
/*eslint eslint-comments/no-use: error */

/* eslint-disable-line */
/* eslint-disable-next-line */
```
